wiki/Tournaments/RDOC/2020/en.md

```diff
@@ -0,0 +1,131 @@
+---
+tags:
+  - RDOC
+  - RDOC2020
+---
+
+# Royal Danish osu!Cup 2020
+
+![RDOC 2020 banner](img/banner.png)
```

Comment:
```suggestion
![RDOC 2020 logo](img/logo.png)
```
Idk we always use logo instead of banner, and if it doesn't containt transparency, may want to do `.jpg` instead.